### PR TITLE
Small updates

### LINKED ISSUE #123: Reset Password screen looks terrible on pixel 3a
Text does not read well on background.
height of textbox is not tall enough for the text.
Screen Shot 2020-02-25 at 9 31 21 PM

### PR DESCRIPTION
**Pull request by esteban737**
Fixes #123

## Description
 Fixed a couple of styling issues on different screens and removed StableAvatar component from all screens. Modified behavior of election system in screen, vote, and candidate picture management

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (specify below)
Style changes and routing changes

## Checklist:

- [x] My code follows the code style of this project found on [Contribution guidelines](https://github.com/SHPEUCF/shpeucfapp/wiki/Contribution-guidelines).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/screens/General/Leaderboard.js [hunk 1]
```diff
@@ -112,9 +111,12 @@ class Leaderboard extends Component {
 								title = { item.firstName[0].concat(item.lastName[0]) }
 							/>	}
```
Comment (idelmr) [inline src/screens/General/Leaderboard.js:112]: Remove tab after `/>`

File: src/screens/General/Leaderboard.js [hunk 2]
```diff
@@ -112,9 +111,12 @@ class Leaderboard extends Component {
...
 							{ item.picture !== ""
-							&& <StableAvatar
-								item = { item }
-							/> }
+							&& <Avatar
+								size = "large"
+								rounded
+								source = {{ uri: item.picture }}
+							/>
+							}
```
Comment (idelmr) [inline src/screens/General/Leaderboard.js:119]: Add ending curly brace on the same line as the closing tag.

File: src/screens/General/More.js [hunk 3]
```diff
@@ -93,11 +99,20 @@ class More extends Component {
...
+			apply,
+			voted
 		} = this.props;
 
-		if (item.title === "Election" && (!election && !apply || !privilege.paidMember))
+		let onPress = Actions[item.screen];
+
+		if (item.title === "Voting") {
+			if (!election || !privilege.paidMember) return null;
+			else if (voted) onPress = (message) => alert(message);
```
Comment (idelmr) [inline src/screens/General/More.js:110]: I would split these conditions in two lines for consistency with the main `if`

File: src/utils/render.js [hunk 4]
```diff
@@ -62,9 +60,12 @@ export const MemberPanel = (user) => {
...
 					{ user.picture !== ""
-					&& <StableAvatar
-						item = { user }
-					/> }
+					&& <Avatar
+						size = "large"
+						rounded
+						source = {{ uri: user.picture }}
+					/>
+					}
```
Comment (idelmr) [inline src/utils/render.js:68]: Add ending curly brace on same line as closing tag.

File: src/utils/render.js [hunk 5]
```diff
@@ -27,16 +27,14 @@ export const verifiedCheckMark = ({ paidMember }) => {
...
-};
-export const StableAvatar = React.memo(userAvatar);
+// const userAvatar = (props) => {
+// 	return (
+// 		<View>
+// 			{ props.children }
+// 		</View>
+// 	);
+// };
+// export const StableAvatar = React.memo(userAvatar);
```
Comment (HanielDiaz) [inline src/utils/render.js:37]: Remove unused comments

**Pull request merged**